spec/features/result_filters/subjects_spec.rb

```diff
@@ -0,0 +1,163 @@
+require "rails_helper"
+
+feature "Subject filter", type: :feature do
+  let(:subject_filter_page) { PageObjects::Page::ResultFilters::SubjectPage.new }
+  let(:results_page) { PageObjects::Page::Results.new }
+
+  let(:subject_areas) do
+    [
+      build(:subject_area, subjects: [build(:subject, :primary, id: 1), build(:subject, :biology, id: 10)]),
+      build(:subject_area, :secondary),
+    ]
+  end
+
+  let(:subject_areas_request) do
+    stub_api_v3_resource(
+      type: SubjectArea,
+      resources: subject_areas,
+      include: [:subjects],
+    )
+  end
+
+  let(:current_recruitment_cycle) { build(:recruitment_cycle) }
```

Comment:
There is a helper -> `stub_results_page_request` that does this down to line 76. We can only use it for now as once the results page has proper functionality we should probably be testing that the correct results appear in these features, in which case, we'll have to do proper set up. There's no point doing that now really as we're not testing the results anyway, just stubbing out call on results to stop it making the network request.